backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/KeyValueHandler.java

```diff
@@ -34,17 +35,30 @@ public KeyValueHandler(Executor executor, FutureJdbi jdbi, String fieldType, Str
     this.jdbi = jdbi;
     this.fieldType = fieldType;
     this.entityName = entityName;
+
+    switch (entityName) {
+      case "ProjectEntity":
+        this.entityIdReferenceColumn = "project_id";
+        break;
+      case "ExperimentRunEntity":
+        this.entityIdReferenceColumn = "experiment_run_id";
+        break;
+      default:
+        throw new InternalErrorException("Invalid entity name");
```

Comment:
Could you add the given entityName to this exception message? It'll help when debugging! Ex. "Invalid entity name: happyBirthdayEntity"